synchronicity/synchronizer.py

```diff
@@ -103,6 +108,8 @@ def __init__(
         multiwrap_warning=False,
         async_leakage_warning=True,
     ):
+        self.loop_delay_monitor_period = 1.0  # seconds
+        self.loop_delay_monitor_threshold = 1.0  # seconds
```

Comment:
Should these be public attributes?

Reply:
That's the idea, so that whatever library that sets up the synchronicity loop can adjust it. But now that I think about it, it's probably better to add them to the synchronizer constructor as optional arguments and keep them private, since changing the period after the event loop thread has started won't have any effect...